lib/src/nonJvmMain/kotlin/cash/z/ecc/android/random/SecureRandom.kt

```diff
@@ -0,0 +1,12 @@
+package cash.z.ecc.android.random
+
+import kotlin.random.Random
+
+actual class SecureRandom {
+
+    actual fun nextBytes(bytes: ByteArray){
```

Comment:
js would be easy enough as you said. will probably have to use cinterop with these system apis: https://libsodium.gitbook.io/doc/generating_random_data

Reply:
Might be worth making a separate library to only do secure random. I have needs for it in my own project haha

Reply:
or if you had a c library in mind to use with cinterop, checkout how I integrated building a c library for each target in my build.gradle for my project [secretk](https://github.com/luca992/secretk)

Reply:
Alright, libsodium docs mention they internally use:
```
On Windows systems, the RtlGenRandom() function is used.
On OpenBSD and Bitrig, the arc4random() function is used.
On recent FreeBSD and Linux kernels, the getrandom system call is used.
On other Unices, the /dev/urandom device is used.
```
So just using those system apis probably would be easy enough

Reply:
checkout [132da15](https://github.com/zcash/kotlin-bip39/pull/14/commits/132da15beeeeaa4ed32ecff3fa3e228b59c42919)

implemented using:
java secure random for jvm
/dev/urandom for native
crypto.getRandomValues for js

Reply:
What about this SecureRandom for iOS? https://github.com/chRyNaN/krypt/blob/develop/docs/krypt-csprng/krypt-csprng/com.chrynan.krypt.csprng/-secure-random/index.md

Reply:
We do not currently plan on introducing third party dependencies to our bip-39 library.

Reply:
Apple already has API’s for generating secure random data, and the existing native implementation done by Luca is probably sufficient to not need the Apple-specific API.

The hold up on merging this pull request is that it needs to be divided into smaller PRs that we can review. As one large PR, it will take a very long time to get through our review process since multiple security experts in different platforms need to be involved.

Reply:
@ccjernigan I already published my own fork to my own repo so this really isn't my top priority atm. But I might be able to break this pr up as you requested soon. Has anything changed with how you want it done since: https://github.com/zcash/kotlin-bip39/pull/14#issuecomment-1110242959

Reply:
I saw your version got published a while back, and I'm glad you were able to leverage our project to help your own efforts move forward 🎉.

What I wrote before is still pretty accurate.  A few minor notes since then:
 - We had another contributor move the tests from jvmTest to commonTest, so that part is done already.
 - I'd suggest starting with the JVM-only multiplatform refactor.  It should effectively be a no-op, and I should be able to review/merge that very quickly.
 - For the native/JS targets, it might be faster to split that into a PR for each (one for native, one for JS) because then our team can review each independently.

Reply:
JVM-only multiplatform refactor done #158

Reply:
Thanks—I'm doing the review for the JVM-only changes ASAP.

There's a new JetBrains blog post relevant for which native targets we choose to configure:  https://blog.jetbrains.com/kotlin/2023/02/update-regarding-kotlin-native-targets/